docker-entrypoint.sh

```diff
@@ -23,7 +23,19 @@ if [ "$1" = jetty.sh ]; then
 fi
 
 if ! command -v -- "$1" >/dev/null 2>&1 ; then
-	set -- java "-Djava.io.tmpdir=$TMPDIR" -jar "$JETTY_HOME/start.jar" "$@"
+	set -- java -jar "$JETTY_HOME/start.jar" "$@"
+fi
+
+if [ -n "$TMPDIR" ] ; then
```

Comment:
Since `TMPDIR` is set in the Dockerfile, this is basically always going to be true unless someone passes `-e TMPDIR=`, which seems quite unlikely.

In addition to that observation, I also feel like we may as well clean up some of the `jetty.sh`-related stuff in the `Dockerfile` now. `JETTY_STATE` no longer has any effect, since we are no longer passing `jetty.state` to the `start.jar` command. `JETTY_RUN` also has no effect, since all it was being used for previously was the `JETTY_PID` functionality in `jetty.sh`.